### DISCUSSION TITLE
How to stop it from auto adjusting delay?

### DISCUSSION THREAD

**Discussion by laichunguang1** [victornpb/undiscord#443] [1. Questions & Answers]
Hi guys. I'm using this tool now, which is really handy.
But thing made me upset is, it always auto adjusts the delay I set(which is always faster than my setting) after detecting response from API.

I want to use it with a safe way, so that I set delay to a relatively long length, but it doesn't stop atuo adjusting.

How I can solve this?
Thank you.

**Comment by abigailwillow** [comment]


**Reply by laichunguang1** [reply]
Thank you so much! It will be helpful.
